sync-upgrade-guide.adoc

```diff
@@ -0,0 +1,140 @@
+sync.globalHandleCreate(function(dataset_id, data, callback, meta_data) {});
+// >=7.0.0
+sync.handleCreate(function(dataset_id, data, meta_data, callback) {});
+sync.globalHandleCreate(function(dataset_id, data, meta_data, callback) {});
+
+
+// <7.0.0
+sync.handleRead(function(dataset_id, uid, callback, meta_data) {});
+sync.globalHandleRead(function(dataset_id, uid, callback, meta_data) {});
+// >=7.0.0
+sync.handleRead(function(dataset_id, uid, meta_data, callback) {});
+sync.globalHandleRead(function(dataset_id, uid, meta_data, callback) {});
+
+
+// <7.0.0
+sync.handleUpdate(function(dataset_id, uid, data, callback, meta_data) {});
+sync.globalHandleUpdate(function(dataset_id, uid, data, callback, meta_data) {});
+// >=7.0.0
+sync.handleUpdate(function(dataset_id, uid, data, meta_data, callback) {});
+sync.globalHandleUpdate(function(dataset_id, uid, data, meta_data, callback) {});
+
+
+// <7.0.0
+sync.handleDelete(function(dataset_id, uid, callback, meta_data) {});
+sync.globalHandleDelete(function(dataset_id, uid, callback, meta_data) {});
+// >=7.0.0
+sync.handleDelete(function(dataset_id, uid, meta_data, callback) {});
+sync.globalHandleDelete(function(dataset_id, uid, meta_data, callback) {});
+
+
+// <7.0.0
+sync.handleCollision(function(dataset_id, hash, timestamp, uid, pre, post, meta_data) {});
+sync.globalHandleCollision(function(dataset_id, hash, timestamp, uid, pre, post, meta_data) {});
+// >=7.0.0  Adding a callback to the end of the arguments
+sync.handleCollision(function(dataset_id, hash, timestamp, uid, pre, post, meta_data, callback) {});
+sync.globalHandleCollision(function(dataset_id, hash, timestamp, uid, pre, post, meta_data, callback) {});
+
+
+// <7.0.0
+sync.listCollisions(function(dataset_id, callback, meta_data) {});
+sync.globalListCollisions(function(dataset_id, callback, meta_data) {});
+// >=7.0.0
+sync.listCollisions(function(dataset_id, meta_data, callback) {});
+sync.globalListCollisions(function(dataset_id, meta_data, callback) {});
+
+
+// <7.0.0
+sync.removeCollision(function(dataset_id, collision_hash, callback, meta_data) {});
+sync.globalRemoveCollision(function(dataset_id, collision_hash, callback, meta_data) {});
+// >=7.0.0
+sync.removeCollision(function(dataset_id, collision_hash, meta_data, callback) {});
+sync.globalRemoveCollision(function(dataset_id, collision_hash, meta_data, callback) {});
+```
+
+== Logger changes
+
+The `logLevel` option that can be passed into `fh.sync.init()` is no longer available.
+By default, the Sync Server will not log anything.
+All logging is done using the [debug](https://www.npmjs.com/package/debug) module.
+If you want log output from the Sync Server, you can set the `DEBUG` environment variable.
+For example:
+
+```shell
+DEBUG=fh-mbaas-api:sync
+```
+
+To see all logs from the entire SDK, you can use
+```shell
+DEBUG=fh-mbaas-api:*
+```
+
+All other environment variables & behaviour of the `debug` module can be used if needed.
+
+== Behaviour differences
```

Comment:
given that this requires code changes, perhaps move it above the `Logger Changes` section as that one doesn't require any code changes?